scripts/west_commands/runners/jlink.py

```diff
@@ -14,6 +14,7 @@
 import subprocess
 import sys
 import tempfile
+import psutil
```

Comment:
In addition to what @henrikbrixandersen said, this is not how we handle third party imports in the runners.

See how `intelhex` is handled in nrfjprog.py for what you should do instead.